corruption_t5.py

```diff
@@ -515,7 +545,7 @@ def corrupt_for_vanilla_t5(
     # TODO: Fix the logic below given that the attention mask consists of a list of tensors of rank 2,
     #  and not a single tensor of rank 3.
     batch['attention_mask'] = torch.asarray(batch['attention_mask'])
-
+    random.seed(seed)
```

Comment:
Drop this here, and add it to the appropriate test function

Reply:
Why? since I am using `random` for my seeding, unless I do it in this instance the random calls will not be deterministic.

Reply:
See my answer above

Reply:
See CorruptionTest in `corrruption_t5_test.py`. You'll see there that if you set the seed at the start of the test, you will get deterministic behavior. In those unit tests, we compare corruption functions on specific tensor inputs, and expect to get back specific tensor outputs.